contrib/completion/bash/docker

```diff
@@ -1597,8 +1598,18 @@ _docker_service_rm() {
 _docker_service_ls() {
 	case "$cur" in
 		-*)
+			COMPREPLY=( $( compgen -W "--help --filter --quiet" -- "$cur" ) )
+			;;
+	esac
+}
+
+_docker_service_scale() {
+	case "$cur" in
+		-*)
 			COMPREPLY=( $( compgen -W "--help" -- "$cur" ) )
 			;;
+		*)
+			COMPREPLY=( $(compgen -S "=" -o nospace -W "$(__docker_services $1)" -- "$cur") )
```

Comment:
@mgoelzer you have to use `compopt -o nospace` as a separate command. But this does not work on Macs, so the idiom used here is `__docker_nospace`, see https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L980 for an example.